aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -300,6 +349,73 @@ TORCH_PRECOMPUTE_META_FUNC(index_add)
   return TORCH_PRECOMPUTE_STRUCT(index_add)().set_dim(dim);
 }
 
+static void build_index_op(
+    TensorIteratorBase& iter,
+    const at::native::AdvancedIndex& info,
+    const Tensor& result) {
+  // 'TensorIterator' needs to own the things comming from 'info', since
+  // 'info' will be destroyed after the META function.
```

Comment:
Should info be saved in the struct for impl? We can do this with precompute right?

Reply:
Hmm... Is it okay to use `AdvancedIndex` type (holds both a `Tensor` and a `std::vector<Tensor>`) with precompute? I thought precompute wasn't supposed to be used for `Tensor`.